cluster/src/main/java/org/apache/iotdb/cluster/ClientMain.java

```diff
@@ -203,7 +203,7 @@ private static void doQuery(boolean noOption, CommandLine commandLine)
         queryPorts = parseIntArray(commandLine.getOptionValue(PARAM_QUERY_PORTS));
       }
       if (queryPorts == null) {
-        queryPorts = new int[] {55560, 55561, 55562};
+        queryPorts = new int[] {port, port + 2, port + 4};
```

Comment:
It seems that queryPorts should be `{port, port + 1, port + 2}`?

Reply:
ok